packages/paste-design-tokens/tokens/global/border-color.yml

```diff
@@ -146,18 +146,24 @@ props:
     comment: Weakest primary border color
 
   # status borders
-  color-border-neutral-weak:
-    value: "{!palette-blue-40}"
-    comment: Weak neutral border color
   color-border-neutral:
     value: "{!palette-blue-60}"
     comment: Neutral border color
+  color-border-neutral-weak:
```

Comment:
Since these were added w/ the intention of using them for a border, and are not used at all, I might take these design tokens out, depending on what design says to do.

Reply:
consulted w/ design and they indicated they'd like these to stay, so they're stayin'!